xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3314,6 +3314,16 @@ void CVideoPlayer::SetSubtitleVisibleInternal(bool bVisible)
   CServiceBroker::GetDataCacheCore().SignalSubtitleInfoChange();
 }
 
+void CVideoPlayer::SetSubtitleVerticalPosition(int value, bool save)
```

Comment:
Looking at the renderManager calls in this file I'm unsure this particular usage is correct.  To me subtitle vertical position is something that belongs to the player context. We already have:

```
void CVideoPlayer::SetVideoSettings(CVideoSettings& settings)
{
  m_processInfo->SetVideoSettings(settings);
  m_renderManager.SetVideoSettings(settings);
  m_renderManager.SetDelay(static_cast<int>(settings.m_AudioDelay * 1000.0f));
  ... 
}
```

Can't this be saved in the player object and just passed to the render manager like all the other player settings?

Reply:
i have implemented to CVideoSettings try to see